<ui version="4.0" >
 <class>PhysicsWidget</class>
 <widget class="QWidget" name="PhysicsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Physics Settings</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QFrame" name="frame" >
     <property name="minimumSize" >
      <size>
       <width>321</width>
       <height>23</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="m_enabled" >
        <property name="text" >
         <string>Physics Enabled</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_debug" >
        <property name="text" >
         <string>Render Shapes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_resetWorld" >
        <property name="minimumSize" >
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
        <property name="text" >
         <string>Reset Physics World</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Mass:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="m_mass" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="specialValueText" >
        <string>Static</string>
       </property>
       <property name="maximum" >
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_kinematic" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>Kinematic</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Restitution:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="m_restitution" >
       <property name="decimals" >
        <number>3</number>
       </property>
       <property name="maximum" >
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>Static Friction:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="m_staticFriction" >
       <property name="minimum" >
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum" >
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.500000000000000</double>
       </property>
       <property name="value" >
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Collision Shape:</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="3" column="1" >
    <widget class="QStackedWidget" name="m_shapeWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="m_boxWidget" >
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>1</number>
       </property>
       <property name="topMargin" >
        <number>1</number>
       </property>
       <property name="rightMargin" >
        <number>1</number>
       </property>
       <property name="bottomMargin" >
        <number>1</number>
       </property>
       <property name="horizontalSpacing" >
        <number>4</number>
       </property>
       <property name="verticalSpacing" >
        <number>1</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Depth:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QDoubleSpinBox" name="m_width" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="minimum" >
          <double>0.001000000000000</double>
         </property>
         <property name="maximum" >
          <double>5000.000000000000000</double>
         </property>
         <property name="value" >
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QDoubleSpinBox" name="m_height" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="minimum" >
          <double>0.001000000000000</double>
         </property>
         <property name="maximum" >
          <double>5000.000000000000000</double>
         </property>
         <property name="value" >
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QDoubleSpinBox" name="m_depth" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="minimum" >
          <double>0.001000000000000</double>
         </property>
         <property name="maximum" >
          <double>5000.000000000000000</double>
         </property>
         <property name="value" >
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_sphereWidget" >
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>1</number>
       </property>
       <property name="topMargin" >
        <number>1</number>
       </property>
       <property name="rightMargin" >
        <number>1</number>
       </property>
       <property name="bottomMargin" >
        <number>1</number>
       </property>
       <property name="horizontalSpacing" >
        <number>4</number>
       </property>
       <property name="verticalSpacing" >
        <number>1</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Radius:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QDoubleSpinBox" name="m_radius" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="minimum" >
          <double>0.001000000000000</double>
         </property>
         <property name="maximum" >
          <double>5000.000000000000000</double>
         </property>
         <property name="value" >
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>61</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_cylinderWidget" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label_9" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Radius:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="m_cylRadius" >
         <property name="decimals" >
          <number>4</number>
         </property>
         <property name="minimum" >
          <double>0.001000000000000</double>
         </property>
         <property name="maximum" >
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="m_cylHeight" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="minimum" >
          <double>0.001000000000000</double>
         </property>
         <property name="maximum" >
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_meshWidget" />
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QComboBox" name="m_shape" >
     <item>
      <property name="text" >
       <string>Box</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Sphere</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Cylinder</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Mesh</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
